### PR TITLE
Update Slack integration prereqs

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Update prereq in [Slack notifications](https://docs.getdbt.com/docs/deploy/job-notifications#slack-notifications) to include more info about channels

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).

### CODE REVIEW

File: website/docs/docs/deploy/job-notifications.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Any account admin can edit the Slack notifications but they'll be limited to con
 ### Prerequisites 
 - You must be an administrator of the Slack workspace. 
 - You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
+- The notification channel must be a public channel. The integration only supports public channels in the Slack workspace.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/job-notifications.md]:
```suggestion
- The notification channel must be a public channel. The integration only supports public channels that are in the Slack workspace. 
```

File: website/docs/docs/deploy/job-notifications.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ Any account admin can edit the Slack notifications but they'll be limited to con
 ### Prerequisites 
 - You must be an administrator of the Slack workspace. 
 - You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
+- The notification channel must be a public channel. The integration only supports public channels that are in the Slack workspace.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/job-notifications.md]:
```suggestion
- Make sure the notification channel is a public channel. The integration only supports public channels that are in the Slack workspace. 
```

File: website/docs/docs/deploy/job-notifications.md [hunk 3]
```diff
@@ -52,6 +52,7 @@ Any account admin can edit the Slack notifications but they'll be limited to con
 ### Prerequisites 
 - You must be an administrator of the Slack workspace. 
 - You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
+- Make sure the notification channel is a public channel. The integration only supports public channels that are in the Slack workspace.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/job-notifications.md]:
```suggestion
- Make sure the notification channel (where you want to receive alerts) is a public channel. The integration only supports public channels in the Slack workspace. 
```

File: website/docs/docs/deploy/job-notifications.md [hunk 4]
```diff
@@ -52,6 +52,7 @@ Any account admin can edit the Slack notifications but they'll be limited to con
 ### Prerequisites 
 - You must be an administrator of the Slack workspace. 
 - You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
+- Make sure the notification channel (where you want to receive alerts) is a public channel. The integration only supports public channels in the Slack workspace.
```
Comment (matthewshaver) [inline website/docs/docs/deploy/job-notifications.md:55]: We could probably cut this down to just
```suggestion
- The integration only supports public channels in the Slack workspace. 
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**